### PR TITLE
rewrite the arm

### LINKED ISSUE #277: Arm rework

### LINKED ISSUE #290: delete stale autos
autos that don't work should get removed. they need to get remade anyways (per alice)

### PR DESCRIPTION
**Pull request by IanSzalai**
closes #277 
closes #290

### CODE REVIEW

**Comment by IanSzalai** [comment]
i think everything is here but im not 100% sure

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -115,105 +115,55 @@ private void configureBindings() {
...
     }));
 
+    // high cube
+    conNumpad.btn_4.onTrue(Commands.runOnce(() -> {
+      subArm.setDesiredNode(2);
+    }));
     // high cube
     conNumpad.btn_6.onTrue(Commands.runOnce(() -> {
       subArm.setDesiredNode(1);
     }));
```
Comment (TaylerUva) [inline src/main/java/frc/robot/RobotContainer.java:165]: Are these both high cube?

File: src/main/java/frc/robot/commands/IntakeFloor.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  Arm subArm;
+  Intake subIntake;
+
+  public IntakeFloor(Arm subArm, Intake subIntake) {
+    this.subArm = subArm;
+    this.subIntake = subIntake;
+
+    addCommands(
+        Commands.runOnce(() -> subArm.setGoalState(ArmState.FLOOR_INTAKE)),
+        Commands.waitUntil(() -> subArm.getCurrentState() == ArmState.FLOOR_INTAKE),
```
Comment (TaylerUva) [inline src/main/java/frc/robot/commands/IntakeFloor.java:24]: Does this method exist?

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (IanSzalai) [reply src/main/java/frc/robot/RobotContainer.java:165]: no, node 1 is cone. comment is incorrect

Comment (IanSzalai) [reply src/main/java/frc/robot/RobotContainer.java]: resolved in latest commit

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**